src/components/Books.js

```diff
@@ -1,7 +1,6 @@
-/* eslint-disable import/extensions */
 import React from 'react';
-import MyBook from './bookItem.js';
-import Form from './bookForm.js';
+import MyBook from './BookItem';
+import Form from './BookForm';
 
 function Books() {
   const Books = [
```

Comment:
- [ ] I'm not able to inline the number 19 > 24,..CSS class naming convention: should use hyphen-case for CSS class names is a convention widely used in HTML and CSS. Following this convention makes your code more consistent and easier to read for other developers. kindly fix this :wink: